app/services/katello/event_daemon.rb

```diff
@@ -0,0 +1,55 @@
+module Katello
+  class EventDaemon
+    def self.runnable?
+      if defined?(Rails::Console)
+        Rails.logger.info("Not starting Katello Event Daemon in the Rails Console")
+        return false
+      end
+
+      if $PROGRAM_NAME.match(/spring/)
+        Rails.logger.info("Spring is running in the environment, not starting Katello Event Daemon")
+        return false
+      end
+
+      if Rails.env == 'production' && $PROGRAM_NAME.match(/dynflow/)
+        Rails.logger.info("This appears to be the Dynflow process, not starting Katello Event Daemon")
+        return false
+      end
```

Comment:
what about a random rake task, would it get started in those?

Reply:
See few lines down, checking `Foreman.in_rake?` there

Reply:
ahh!!! :+1: